static/templates/base/index.html

```diff
@@ -23,13 +23,16 @@
 </head>
 
 <body ng-controller="HomeController as ctrl">
-<div id="appbar" layout="row" ng-controller="NavbarController as navCtrl">
+
+
+<div id="appbar" layout="row" ng-controller="NavbarController as navCtrl" ng-if="isLoggedIn">
     <md-toolbar class="md-lightblue-theme">
         <div class="md-toolbar-tools" flex>
-            <div flex layout="row" layout-align="center">
-                <h1 ng-if="isLoggedIn"  layout-align="left"><a href="/">DAEMO</a></h1>
-                <h1 ng-if="!isLoggedIn" layout-align="left">Next Generation Crowdsourcing</h1>
-
+            <div flex="35" layout="row" layout-align="left">
+                <h1 layout-align="left"><a href="/">DAEMO</a></h1>
+            </div>
+            <div flex layout="row" layout-align="right">
+                <ng-include ng-if="isLoggedIn" src="'/static/templates/base/top_navigation.html'"></ng-include>
```

Comment:
we'll do this when we work on design, this is a basic implementation as in the description.